### DISCUSSION TITLE
Circles intro sequence should use the slider background instead of the triangles one

### DISCUSSION THREAD

**Discussion by N0Y0U2114** [ppy/osu#30957] [Ideas] [answered]
Pretty self explanitory. In stable it uses this dynamic background (called "slider" on merch)
![image](https://github.com/user-attachments/assets/18d9e53f-03a5-4053-9bc7-d6e21870ac42)
In lazer it uses the same background as the triangles intro sequence
![image](https://github.com/user-attachments/assets/580d5fe3-a905-445f-85bc-e380ca715973)

It should use the same one as stable to complete the trifecta of intro sequences.

**Comment by N0Y0U2114** [comment, marked as answer]
https://github.com/ppy/osu-resources/pull/353 and https://github.com/ppy/osu/pull/31730

**Comment by peppy** [comment]
I guess we can do this... but it wasn't really the direction I was thinking. The stable backgrounds have historically changed every few years, and they aren't really attached to a certain "skin".

**Reply by N0Y0U2114** [reply]
Are there plans to continue to change the background given that stable is at the end of its gaining new features life? I would think that the new backgrounds would continue through lazer
